extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -246,3 +252,9 @@ export class NetCoreTool {
 			});
 	}
 }
+
+export class DotNetError extends Error {
+	constructor(message?: string) {
```

Comment:
You shouldn't need this constructor

Reply:
Whoops; my mistake.  Charles is correct.

Reply:
Done.